### PR TITLE
Minor fixes and changes

### PR DESCRIPTION
**Pull request by conman180**
1. remove pitch down action when slower than stall speed.(would cause planes to constantly move on ground)
2. sensitivity adjustments to make very slow planes more controllable. Smoothed out sensitivity transition between speed settings
3. lengthened time requirement for shoot down timer.

### CODE REVIEW

File: src/main/java/com/flansmod/common/driveables/FlightControllerNew.java [hunk 1]
```diff
@@ -111,18 +111,20 @@ public void SetAxes(EntityPlane plane)
 			
 			// Plane mode.
 			// main sensitivity.
-			if (speed < 0.5F) {
+			if (speed < 0.25F) {
```
Comment (GoldSloth) [inline src/main/java/com/flansmod/common/driveables/FlightControllerNew.java:114]: It looks like this is no longer using takeoff speed, is that intended?

File: src/main/java/com/flansmod/common/driveables/FlightControllerNew.java [hunk 2]
```diff
@@ -111,18 +111,20 @@ public void SetAxes(EntityPlane plane)
...
+				sensitivityAdjust = (2 * speed) - 1;
+			} else if (speed < 3F) {
 				sensitivityAdjust = 1.5F - (speed/2);
 			} else if (speed >= 2.0F && type.supersonic == true) {
 				sensitivityAdjust = 2.5F - (speed/2);
 			}
 
 			// If 0<speed<0.625, we'll call this taxi mode. Yaw sensitivity will work differently to main sensitivity.
-			if (groundSpeed < type.takeoffSpeed) {
+			if (groundSpeed < 0.5F) {
```
Comment (GoldSloth) [inline src/main/java/com/flansmod/common/driveables/FlightControllerNew.java:127]: Same here, is removing takeoff speed needed?

File: src/main/java/com/flansmod/common/driveables/FlightControllerNew.java [hunk 3]
```diff
@@ -365,43 +364,39 @@ public void PlaneModeFly(EntityPlane plane)
 	          plane.axes.rotateLocalRoll(5.0F); 
 	        doomsday++;
 	        if (plane.axes.getPitch() < 35.0F)
-	          plane.axes.rotateLocalPitch(-0.05F * doomsday); 
-	        if (plane.isEngineActive())
+	          plane.axes.rotateLocalPitch(-0.05F * doomsday);
```
Comment (GoldSloth) [inline src/main/java/com/flansmod/common/driveables/FlightControllerNew.java:367]: Not sure if this is intended too, removing if (plane.isEngineActive) to damage the part.

Comment (conman180) [reply src/main/java/com/flansmod/common/driveables/FlightControllerNew.java:367]: Answer to q 1: having the sensitivity being related to takeoff speed was causing weird things to hppen and i just reverted back to the original sensitivities with the exception of being btw 0.25 and 0.5. creates a better gradient as you speed up.

Answer to Q2: when the engine was active, the timer would tick extremely fast and give almost no time before exploding, when non critical part were destroyed. shooting a wing off should make it explode right away.

Comment (GoldSloth) [reply src/main/java/com/flansmod/common/driveables/FlightControllerNew.java:367]: That makes sense, thanks for answering those questions. I'll merge it later today hopefully.

**Pull request merged**